lib/runner.js

```diff
@@ -37,6 +37,10 @@ class Runner extends EventEmitter {
     this.reporter = null;
   }
 
+  runTodo(active = true) {
+    this.options.runTodo = runTodo;
```

Comment:
```suggestion
    this.options.active = active;
```

Reply:
active implies that this whole Runner is working or not, but this option only concerns todo tests.

Reply:
Sorry
```suggestion
    this.options.runTodo = active;
```